src/ApiTestCase.php

```diff
@@ -376,6 +382,26 @@ private function assertSourceExists($source)
     }
 
     /**
+     * @throws \RuntimeException
+     */
+    private function assertResponseLoaderIsAvailable()
+    {
+        if (null === $this->responseLoader) {
+            throw new \RuntimeException('Something went wrong and response loader did not set properly, if you have overridden setUp method check it!');
```

Comment:
`response loader has not been set properly. If you have overridden setUp method, check it!`?